lib/y2_consoletest.pm

```diff
@@ -1,7 +1,9 @@
 # Base for YaST tests.  Switches to text console 2 and uploady y2logs
 
-package console_yasttest;
-use base 'y2logsstep';
+package y2_consoletest;
+use base 'y2_module_basetest';
+use y2_installbase qw(save_upload_y2logs save_system_logs save_strace_gdb_output);
+use y2_installbase;
```

Comment:
y2_installbase is imported twice. Is there some reason for this?

Reply:
i dont know why this is there. on my computer this is just the first line